regression/cbmc/gcc_builtin_add_overflow/main.c

```diff
@@ -0,0 +1,80 @@
+#include <limits.h>
+#include <assert.h>
+
```

Comment:
Tests look good, but could we add a few more cases? In particular, these tests only cover some of the "easily optimised" cases (e.g value of 1, adding/subtracting to MAX/MIN), but it would be great to see some tests that also cover "mid-range value x + mid-range value y == overflow" and  "slightly smaller mid-range value x + mid-range value y != overflow" cases.

Reply:
@chrisr-diffblue I’ve added some more cases to that end